app/src/androidTest/java/org/mozilla/tv/firefox/FirefoxTestApplication.kt

```diff
@@ -11,8 +11,23 @@ import org.mozilla.tv.firefox.utils.ServiceLocator
 class FirefoxTestApplication : FirefoxApplication() {
 
     override fun createServiceLocator() = object : ServiceLocator(this) {
-        override val pocketRepo = PocketRepoFaker.fakedPocketRepo
+        override val pocketRepo by lazy { TestProvider.pocketVideoRepo ?: super.pocketRepo }
```

Comment:
Why are we mutating individual items in the serviceLocator? It feels more straightforward, and less boilerplate, to swap in an entirely new ServiceLocator where `EachTestServiceLocator : ServiceLocator`.